.github/workflows/CI.yml

```diff
@@ -218,37 +237,51 @@ jobs:
             mingw-w64-x86_64-opus 
             mingw-w64-x86_64-x265
             mingw-w64-x86_64-boost 
+            mingw-w64-x86_64-nsis
             git 
             yasm 
             nasm 
             diffutils 
             make
       - name: Build Windows
         shell: msys2 {0}
-        run: |
-          mkdir sunshine-windows-build && cd sunshine-windows-build
+        run: |   
+          mkdir sunshine-windows-build
+          cd sunshine-windows-build
           cmake -DCMAKE_BUILD_TYPE=Release -DSUNSHINE_ASSETS_DIR=assets -G "MinGW Makefiles" ..
           mingw32-make -j2
-      - name: Package Windows
+      - name: Create packages
+        shell: msys2 {0}
         run: |
           cd sunshine-windows-build
-          del ..\assets\apps_linux.json
-          7z a sunshine-windows.zip ..\assets
-          7z a sunshine-windows.zip sunshine.exe
-          7z a sunshine-windows.zip tools\dxgi-info.exe
-          7z a sunshine-windows.zip tools\audio-info.exe
-          7z a sunshine-windows.zip tools\sunshinesvc.exe
-          7z a sunshine-windows.zip ..\tools\install-service.bat
-          7z a sunshine-windows.zip ..\tools\uninstall-service.bat
-          cd ..
-          mkdir artifacts
-          move "sunshine-windows-build\sunshine-windows.zip" "artifacts"
-      - name: Upload Artifacts
+          cpack
+
+          mkdir -p artifacts/standalone
+          mkdir -p artifacts/web
+          mkdir -p artifacts/shaders/directx
+          
+          # Installers
+          mv Sunshine__.exe artifacts/sunshine-windows-installer.exe
+          
+          # Standalone
+          mv ../assets/web artifacts/web
+          mv tools artifacts/tools
+          mv ../assets/shaders/directx artifacts/shaders/directx
+          mv ../assets/apps_windows.json artifacts/apps_windows.json
+          mv ../assets/sunshine.conf artifacts/sunshine.conf
+          mv sunshine.exe artifacts/sunshine.exe
```

Comment:
This won't work like this. We need only 2 files in the artifacts folders.

- installer exe
- zip (portable version as before)

This is because every file single in the assets directory will be uploaded as a release asset.

My code snip before should work (except the mkdir artifacts needed to be moved)

Reply:
This should be pretty close https://github.com/SunshineStream/Sunshine/pull/139#discussion_r860248861